src/pybind/mgr/crash/module.py

```diff
@@ -149,6 +181,8 @@ def do_post(self, cmd, inbuf):
             metadata = self.validate_crash_metadata(inbuf)
         except Exception as e:
             return errno.EINVAL, '', 'malformed crash metadata: %s' % e
+        metadata['stack_sig'] = \
+            self.calc_sig(metadata['backtrace'], metadata['assert_msg'])
```

Comment:
assert_msg not always present..
```
2019-11-05T23:49:12.433 INFO:tasks.ceph.mgr.x.smithi143.stderr:2019-11-05T23:49:12.442+0000 7f56ec6fe700 -1 mgr.server reply reply (22) Invalid argument Traceback (most recent call last):
2019-11-05T23:49:12.434 INFO:tasks.ceph.mgr.x.smithi143.stderr:  File "/usr/share/ceph/mgr/mgr_module.py", line 975, in _handle_command
2019-11-05T23:49:12.434 INFO:tasks.ceph.mgr.x.smithi143.stderr:    return self.handle_command(inbuf, cmd)
2019-11-05T23:49:12.434 INFO:tasks.ceph.mgr.x.smithi143.stderr:  File "/usr/share/ceph/mgr/crash/module.py", line 107, in handle_command
2019-11-05T23:49:12.434 INFO:tasks.ceph.mgr.x.smithi143.stderr:    return handler(self, command, inbuf)
2019-11-05T23:49:12.434 INFO:tasks.ceph.mgr.x.smithi143.stderr:  File "/usr/share/ceph/mgr/crash/module.py", line 185, in do_post
2019-11-05T23:49:12.434 INFO:tasks.ceph.mgr.x.smithi143.stderr:    self.calc_sig(metadata['backtrace'], metadata['assert_msg'])
2019-11-05T23:49:12.434 INFO:tasks.ceph.mgr.x.smithi143.stderr:KeyError: ('assert_msg',)
```

Reply:
Oops, will fix

Reply:
Pushed new commit that should address this.